docs/deployment.md

```diff
@@ -0,0 +1,205 @@
+
+The Examples folder in the Postman collection includes an [eCommerce example](https://www.postman.com/interledger/workspace/interledger/folder/22855701-e27838da-dd72-4b5e-9f1e-086ddfa4d098?ctx=documentation) that can be executed one by one. It
+
+1. requests a grant to create an incoming payment on Philip Fry's account
+2. creates an incoming payment on Philip Fry's account
+3. requests a grant to create (and read) a quote and an outgoing payment on Grace Franklin's account)
+4. continues the grant request
+5. creates a quote on Grace Franklin's account
+6. creates an outgoing payment on Grace Franklin's account
+7. fetches the outgoing payment on Grace Franklin's account
+
+Note that one has to go throught the interaction flow after requesting a grant for a quote and outgoing payment.
+
+<video alt="Screen Recoding eCommerce Example" src="./mov/eCom-example.mov" width="560" height="315" controls></video>
+
+### SPSP
+
+Every payment pointer also serves as an SPSP endpoint. A GET request to e.g. `http://localhost:3000/accounts/gfranklin` with `Accept` header `application/spsp4+json` will return an SPSP response with STREAM connection details.
+
+```http
+http GET http://localhost:3000/accounts/gfranklin Host:backend Accept:application/spsp4+json
+
+HTTP/1.1 200 OK
+Connection: keep-alive
+Content-Length: 220
+Content-Type: application/spsp4+json
+Date: Thu, 23 Feb 2023 13:07:24 GMT
+Keep-Alive: timeout=5
+
+{
+    "destination_account": "test.rafiki.viXmy1OVHgvmQakNjX1C6kQMri92DzHeISEv-5VzTDuFhrpsrkDzsq5OO9Lfa9yed0L2RJGC9hA_IX-zztTtTZ87shCYvsQ",
+    "receipts_enabled": false,
+    "shared_secret": "Rz_vudcg13EPs8ehL2drvZFJS1LJ4Y3EltOI60-lQ78"
+}
+
+```
+
+### Shutting down the local environment
+
+```
+// tear down
+pnpm localenv:stop
+
+// delete database volumes (containers must be removed first with e.g. pnpm localenv:stop)
+pnpm localenv:dbvolumes:remove
+```
+
+## Production Environment
+
+The production environment consists of
+
+- `backend`
+- (optional but recommended) `auth`
+
+and the databases
+
+- Tigerbeetle or Postgres (accounting)
+- Postgres (Open Payments resources, auth resources)
+- Redis (STREAM details)
+
+To integrate Rafiki with your own services, view the [integration documentation](./integration.md).
+
+### Running the production environment
+
+Dependencies:
+
+- [Kubernetes](https://kubernetes.io/releases/download/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [helm](https://helm.sh/docs/intro/install/)
+
+TODO: Describe helm charts and how to install them on kubernetes cluster once we have them
+
+```
+// add rafiki repository
+$ helm repo add ...
+
+// install rafiki components
+$ helm install ...
+```
+
+### Environment Variables
+
+#### Backend
```

Comment:
also something happened to table formatting (when looking at the markdown version)